src/Tmds.Ssh/PackingAlgorithm.cs

```diff
@@ -3,18 +3,17 @@
 
 using System;
 using System.Collections.Generic;
-using System.Security.Cryptography;
 
 namespace Tmds.Ssh;
 
-sealed class EncryptionAlgorithm
+sealed class PackingAlgorithms
```

Comment:
Let's keep the old name (since we're using a different name for the openssl key ciphers).